### ISSUE TITLE
return localized names in search responses

### ISSUE THREAD

**Issue by DerDakon** [OpenRailwayMap/OpenRailwayMap#674]
The API only returns the "name" tag, but localized names could be handy, especially when using a localized version of the website.

**Comment by ferrocarta** [comment]
Indeed, or even extend the search to names in any language independently of localization. For example Tokyo station can only be found by entering 東京, but it should also be found by entering Tōkyō, Tokyo or Tokio.
